src/bridge/cockpitpackages.c

```diff
@@ -850,7 +851,9 @@ handle_packages (CockpitWebServer *server,
       goto out;
     }
 
-  bytes = cockpit_web_response_negotiation (filename, package->paths, &chosen, &error);
+  languages = cockpit_web_server_parse_languages (headers, NULL);
+
+  bytes = cockpit_web_response_negotiation (filename, package->paths, languages[0], &chosen, &error);
```

Comment:
I don't think so. At least not for now. I've found no way to make this predictable. If we want to add this in the future then that shouldn't be a problem.